agreement/views/agreement.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0"?>
+<!--
+  © 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<odoo>
+
+
+<record id="agreement_form" model="ir.ui.view">
+    <field name="name">agreement.form</field>
+    <field name="model">agreement</field>
+    <field name="arch" type="xml">
+        <form string="Agreement">
+            <div class="oe_button_box" name="button_box">
+                <button name="toggle_active" type="object"
+                    class="oe_stat_button" icon="fa-archive">
+                    <field name="active" widget="boolean_button"
+                        options='{"terminology": "archive"}'/>
+                </button>
+            </div>
+            <group name="main">
+                <group name="left">
+                    <field name="partner_id"/>
+                    <field name="name"/>
+                    <field name="signature_date"/>
+                </group>
+                <group name="right">
+                    <field name="start_date"/>
+                    <field name="end_date"/>
+                    <field name="code"/>
+                    <field name="company_id" groups="base.group_multi_company"/>
+                </group>
+            </group>
+        </form>
```

Comment:
Sorry, but in fact notebook is useful. It's a problem for you in this view ?

Thanks